packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/browser-extension-autoinjection",
-  "version": "0.29.0",
```

Comment:
I don't think it's needed to change the version here as this should be done by the release automation.

It might be needed to re-add this module in .release-please-manifest.json to let the release automation know about the last release. But I'm not sure here.

@dyladan  I think you know most about release-please. Do you know if adding to .release-please-manifest.json is required?

Reply:
yes it is required. it is also required to add to `.release-please-manifest.json`

Reply:
Reset the version to 0.29 & added the package to .release-please-manifest.json